setup.py

```diff
@@ -26,7 +26,7 @@
         'ipyleaflet!=0.8.2',
         'pyproj',
         'shapely>=1.6.3,<2.*',
-        'rasterio>=1.0,<1.0.3',
+        'rasterio>=1.0.3.post1,<2.*',
```

Comment:
Why do we need to raise the minimum version? Isn't the new `Window` usage also compatible with 1.0?

Reply:
I did this because rasterio 1.0.3 is broken (it cannot open *.jp2 images). Is it possible to exclude 1.0.3 from `>=1.0.post1,<2.*` range in setup.py?

Reply:
I think `!=1.0.3`, but perhaps it's not worth it.